doc/releases/migration-guide-4.1.rst

```diff
@@ -408,6 +408,12 @@ Bluetooth Host
   makes a device vulnerable for downgrade attacks. If an application still needs to use LE legacy
   pairing, it should disable :kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY` manually.
 
+* The ``CONFIG_BT_SEND_ECC_EMULATION`` Kconfig option has been removed, and the host will now
```

Comment:
I guess we can't use `:kconfig:option:` for removed Kconfigs? :)

Reply:
Correct.